themes/vue/source/js/common.js

```diff
@@ -337,8 +336,14 @@
     function makeHeaderClickable (link) {
       var wrapper = link.querySelector('a')
       wrapper.setAttribute('data-scroll', '')
-      link.parentNode.insertBefore(wrapper, link)
-      wrapper.appendChild(link)
+
+      var nodes = link.childNodes
+      for (var i = 0; i < nodes.length; i++) {
+        var node = nodes[i]
+        if (node !== wrapper) {
+          wrapper.appendChild(node)
+        }
+      }
```

Comment:
I found that just remove these additions here and all the modifications in `page.style` could work (in Safari 11 in my Mac), which means just keeping `setAttribute('data-scroll', '')` only in function `makeHeaderClickable` is OK.